lib/config.js

```diff
@@ -38,6 +38,10 @@ var Config = function(src, options) {
       }
     }, this);
   }
+
+  if (options.public) {
```

Comment:
I'm not sure exactly at which point to overwrite this field. This seemed to work for all the cases I tried, but let me know if this check belongs elsewhere.